src/components/Likes/LikesComponent.jsx

```diff
@@ -1,19 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import '../../styles/Likes.css';
+import { likeDeterminer, styleDeterminer } from '../../services/styleSelector';
+
 
-const styleDeterminer = (like, isAuthenticated) => (like == true && isAuthenticated) ? {
```

Comment:
Hello @IssaIan ,
You should not remove the underline button that was used on the like and dislike of articles feature. It was a requirement.

Reply:
*What you have currently*
![image](https://user-images.githubusercontent.com/30015297/60186756-1ab0e500-9835-11e9-8d96-855d4449f0f1.png)
*What was there initially*
![image](https://user-images.githubusercontent.com/30015297/60186775-23092000-9835-11e9-9619-dc68cfc7934f.png)

Reply:
Made the changes.